Source/JavaScriptCore/wasm/WasmBBQJIT.h

```diff
@@ -2260,7 +2260,7 @@ class BBQJIT {
     BBQCallee& m_callee;
     const FunctionData& m_function;
     const FunctionSignature* m_functionSignature;
-    uint32_t m_functionIndex;
+    FunctionSpaceIndex m_functionIndex;
```

Comment:
Should this be `FunctionCodeIndex` since it's initialized with `FunctionCodeIndex functionIndex` passed to `BBQJIT` constructor?

Reply:
Good catch! I guess if you pass a `FunctionCodeIndex` to a `FunctionSpaceIndex` it will implicitly convert to a `size_t` then to a `uint32_t` then gets constructed. I've added 
```
FunctionCodeIndex(FunctionSpaceIndex) = delete;
``` and visa versa to stop that...